mama/c_cpp/src/c/dqstrategyplugin/dqstrategyplugin.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * mamaPluginTemplate.h
+ *
+ *  Created on: 23 Mar 2015
+ */
+
+#ifndef MAMAPLUGINHOOK_H_
```

Comment:
Header guards are duplicated from the plugin hook example, which is going to cause issues with the build. Should be fixed.